lxml/cssselect.pyi

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, List, Optional, Union
```

Comment:
This stubs seems to be incomplete. Could you add a module getattr (see https://www.python.org/dev/peps/pep-0484/#stub-files) to indicate this?

Reply:
i'm also addings this to the other modue stubs.